docs/source/bird-rr-config.rst

```diff
@@ -0,0 +1,149 @@
+Configuring BIRD as a BGP Route Reflector
+=========================================
+
+These instructions will take you through installing BIRD as a BGP route
+reflector, and updating your other BIRD instances to speak to your new route
+reflector.
```

Comment:
You noticed I didn't include a link then! :p Google search gives loads of these, but not obvious which would be most suitable. A calico blog post on the topic at some point might be the way to go? (with a note to come back and add the link here once it's done)